### DISCUSSION TITLE
Safe way to disable covid plugin after sales started

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2640] [Q&A] [answered]
Hey everyone,

in the recent turn of events, I would like to disable covid certificate scanning. Is it safe to disable the plugin once orders were placed, or do you have to manually remove the "invisible" questions? What's the safe way to do this?

Thanks,
Alex

**Comment by raphaelm** [comment, marked as answer]
You should disable the plugin and delete or disable the question.
